### PR TITLE
GG-22187 Removed Web Console code.

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -140,7 +140,6 @@
         <zkclient.version>0.5</zkclient.version>
```
Comment (nva) [inline parent/pom.xml]: Should be removed:
<swagger2.version>2.9.2</swagger2.version>
<commons.compress.version>1.18</commons.compress.version>
<commons.lang3.version>3.9</commons.lang3.version>

File: parent/pom.xml [hunk 2]
```diff
@@ -1005,20 +1004,6 @@
...
-                                        <exclude>**/web-console/backend/build/**</exclude>
-                                        <exclude>**/web-console/backend/node_modules/**</exclude>
-                                        <exclude>**/web-console/e2e/testcafe/node_modules/**</exclude>
-                                        <exclude>**/web-console/frontend/build/**</exclude>
-                                        <exclude>**/web-console/frontend/node_modules/**</exclude>
-                                        <exclude>**/web-console/frontend/**/*.png</exclude>
-                                        <exclude>**/web-console/frontend/**/*.svg</exclude>
                                         <!-- H2 forked module. -->
                                         <exclude>**/org/h2/**</exclude>
                                         <exclude>src/docsrc/**</exclude>
```
Comment (nva) [inline parent/pom.xml]: Remove

<group>
                                <title>Ignite Web Console</title>
                                <packages>org.apache.ignite.console*</packages>
                            </group>

File: parent/pom.xml (hunk 1 above)
Comment (akuznetsov-gridgain) [reply parent/pom.xml]: Fixed

File: parent/pom.xml (hunk 2 above)
Comment (akuznetsov-gridgain) [reply parent/pom.xml]: Fixed

**Comment by vveider** [comment]
ITDS-646 is done. Final check are needed if necessary.

**Pull request merged**